en/assembly/quick_start_durandal.md

```diff
@@ -0,0 +1,178 @@
+
+
+## GPS + Compass + Buzzer + Safety Switch + LED
+
+Durandal is designed to work well with the *Pixhawk 4 GPS module*, which has an integrated compass, safety switch, buzzer and LED.
+It connects directly to the [GPS port](../flight_controller/durandal.md#gps) using the 10 pin cable.
+
+<!-- Confirm how compass delivery works - separate or in the package? -->
+
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
+
+![Connect compass/GPS to Durandal](../../assets/flight_controller/durandal/connection_gps_compass.jpg) 
+
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle).
+  To disable the safety press and hold the safety switch for 1 second.
+  You can press the safety switch again to enable safety and disarm the vehicle (this can be useful if, for whatever reason, you are unable to disarm the vehicle from your remote control or ground station).
+
+
+
+## Power
+
+You can use a power module or power distribution board to power motors/servos and measure power consumption. 
+
+### PM02 v3 Power Module
+
+The *PM02 v3* power module that can be bundled with *Durandal* has the following characteristics/limits:
+- Max input voltage: 60V
+- Max current sensing: 120A Voltage
+- Current measurement configured for SV ADC Switching regulator outputs 5.2V and 3A max 
+- Weight: 20g 
+- Package includes: 
+  - PM02 board
+  - 6pin MLX cable (1)
+  - 6pin GH cable (1) 
+
+Connect the output of the *Power Module* that comes with the kit to one of the Durandal [POWER](../flight_controller/durandal.md#power) ports using the 6-wire GH cable.
+
+The PM input **2~12S** will be connected to your LiPo battery.
+
+The PM output should be connected to TBD
+
+<!-- 
+
+> **Note** The 8 pin power (+) rail of **MAIN/AUX** is not powered by the power module. 
+  If it will need to be separately powered in order to drive servos for rudders, elevons etc.
+  To do this, the power rail needs to be connected to a BEC equipped ESC or a standalone 5V BEC or a 2S LiPo battery.
+  Be careful with the voltage of servo you are going to use.
+
+
+
+The pinout of *Pixhawk 4*’s power ports is shown below. 
+The CURRENT signal should carry an analog voltage from 0-3.3V for 0-120A as default. 
+The VOLTAGE signal should carry an analog voltage from 0-3.3V for 0-60V as default. 
+The VCC lines have to offer at least 3A continuous and should default to 5.1V. 
+A lower voltage of 5V is still acceptable, but discouraged.
+-->
+
+<!-- Data sheet for AP says 18.1 voltage divider, 36 Amps/volt 
```

Comment:
@bkueng How is the PM configured? The data sheet is for ArduPilot and says set 18.1 voltage divider, 36 Amps/volt. However Pixhawk 4 docs suggest you connect number of cells (see following note). What is the "right way"?

Reply:
OK. I'm going to leave that with you :-)